src/auth/service.py

```
@@ -1,30 +1,181 @@
+        return existing_user, False
+
+    # 2) 동일 email이 다른 provider로 이미 가입되어 있는지 확인
+    email_user = await user_service.get_user_by_email(
+        session=session, email=email
+    )
+
+    if email_user:
+        raise OAuthError(
+            f"이미 {email_user.oauth_provider.value}(으)로 가입된 이메일입니다. "
+            f"{email_user.oauth_provider.value} 로그인을 이용해 주세요."
+        )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)